### PR TITLE
Add a 'payment method details' screen

### LINKED ISSUE #5049: Add a 'payment method details' screen in Bisq client
### Description

Bisq has a section that shows active offers per currency. It is in a screen called 'details'

It would be great if Bisq had a section that shows active offers per payment  method. 

I proposed the current screen called 'details' to be renamed to ' currency details' and a new screen to be added called 'payment method details'

### Benefits

It would be great for Bisq users to see how many deals are available for what particular types of payment methods eg; Zelle, Revolut, TransferWise.

This would help traders see what is currently popular / available. It might prompt traders to create / add payment methods that are popular to their Bisq client. 

It would also be really useful to my role of Payment Maintainer when generating reports of payment method activity. 

#### Version

1.5.4

### Actual behaviour

On Bisq client

Market > Details

### Expected behaviour

On Bisq client there to be two options

Market > Payment Method Details

Market > Currency Details

### Screenshots

Add payment method details

### PR DESCRIPTION
**Pull request by ghost**
Fixes #5049

image

---

image

### CODE REVIEW

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -292,7 +292,8 @@ mainView.version.update=(Update available)
 ####################################################################
 
 market.tabs.offerBook=Offer book
-market.tabs.spread=Details
+market.tabs.spreadCurrency=Currency Details
```
Comment (chimp1984) [inline core/src/main/resources/i18n/displayStrings.properties]: Maybe "Offers by currency" and "Offers by Payment Method" would be better?

**Review by chimp1984 — DISMISSED** [review]
utACK

**Comment by chimp1984** [comment]
That was fast! ;-)

**Review by sqrrm — APPROVED** [review]
utACK

**Pull request merged**